samples/routing/instance-mapping-file/sample.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Mapping endpoint instances using a shared file
+summary: Mapping logical endpoints to physical instances using a shared file
+component: Core
+tags:
+ - Routing
+ - MSMQ
+---
+
+The sample demonstrates to use a file to describe the mapping between logical endpoints and their physical instances. This is important for store-and-forward transports such as MSMQ because the physical topology can differ from the logical one.
```

Comment:
> This is important for store-and-forward transports such as MSMQ because the physical topology can differ from the logical one.

Not sure if that's obvious what you mean, maybe some example? Is that that logically you have a single endpoint, but physically there might be multiple instances? If so, mention it.